snuba/cli/optimize.py

```diff
@@ -8,16 +8,18 @@
 
 
 @click.command()
-@click.option('--clickhouse-server', multiple=True,
```

Comment:
Oops, just noticed this was `multiple=True`, so this won't work as-is. Not sure it makes sense to have `--clickhouse-server` take multiple values but `--clickhouse-port` take a single values, so I'm going to go ahead and remove the loop on `clickhouse_server` below.

Other options:

1.) Revert the changes here and leave as is, so multiple `host:port` values can be provided.
1a.) Allow multiple `host` (with default port) or `host:port` values.
2.) Allow equal length `--clickhouse-server` and `--clickhouse-port` values and zip them to get `(host, port)` pairs, or something similarly awkward. (Please don't make me do this.)